authentication/AuthenticationExample/Models/AuthProvider.swift

```diff
@@ -23,13 +23,28 @@ enum AuthProvider: String {
   case gitHub = "GitHub"
   case yahoo = "Yahoo"
   case facebook = "Facebook"
-  case EmailPassword = "Email & Password Login"
-  case Passwordless = "Email Link/Passwordless"
-  case PhoneNumber = "Phone Number"
-  case Anonymous = "Anonymous Authentication"
-  case Custom = "Custom Auth System"
+  case emailPassword = "Email & Password Login"
+  case passwordless = "Email Link/Passwordless"
+  case phoneNumber = "Phone Number"
+  case anonymous = "Anonymous Authentication"
+  case custom = "Custom Auth System"
 
-  var id: String { rawValue.lowercased().appending(".com") }
+  var id: String {
+    switch self {
+    case .emailPassword:
+      return "password"
+    case .phoneNumber:
+      return "phone"
+    case .passwordless:
+      return "emailLink"
+    case .anonymous:
+      return "anonymous"
+    case .custom:
+      return "custom"
+    default:
+      return rawValue.lowercased().appending(".com")
+    }
+  }
```

Comment:
It may make more sense to use the provider's id as the raw value of the enum and then have a method that transforms the enum into a display string, instead of having your display string back your model type and then transforming the display string into an ID.

Reply:
This makes sense to me. I will adjust! 👍